stories/index.js

```diff
@@ -205,3 +205,11 @@ storiesOf('ChipInput', module)
       addOnBlur
     />
   ))
+  .add('single form field', () => themed(
+    <form>
```

Comment:
Yup. In my use case we use a form specifying an onSubmit handler and no action (which we want, and expect to be triggered by the form's submit button), but in forms where the only input is ChipInput pressing enter adds the chip and the browser automatically triggers the submit.

In the story this behaviour presents as the form submitting to `iframe.html?`.